test/TestProjects/PetStore-Typespec/Generated/PetStoreClient.cs

```diff
@@ -103,51 +103,34 @@ public virtual Response Delete(int petId, RequestContext context = null)
         /// <summary> Returns a pet. Supports eTags. </summary>
         /// <param name="petId"> The id of pet. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<Pet>> ReadValueAsync(int petId, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<Pet>> ReadAsync(int petId, CancellationToken cancellationToken = default)
         {
-            using var scope = ClientDiagnostics.CreateScope("PetStoreClient.ReadValue");
-            scope.Start();
-            try
-            {
-                RequestContext context = FromCancellationToken(cancellationToken);
-                Response response = await ReadAsync(petId, context).ConfigureAwait(false);
-                return Response.FromValue(Pet.FromResponse(response), response);
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Response response = await ReadAsync(petId, context).ConfigureAwait(false);
+            return Response.FromValue(Pet.FromResponse(response), response);
         }
 
         /// <summary> Returns a pet. Supports eTags. </summary>
         /// <param name="petId"> The id of pet. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual Response<Pet> ReadValue(int petId, CancellationToken cancellationToken = default)
+        public virtual Response<Pet> Read(int petId, CancellationToken cancellationToken = default)
         {
-            using var scope = ClientDiagnostics.CreateScope("PetStoreClient.ReadValue");
-            scope.Start();
-            try
-            {
-                RequestContext context = FromCancellationToken(cancellationToken);
-                Response response = Read(petId, context);
-                return Response.FromValue(Pet.FromResponse(response), response);
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Response response = Read(petId, context);
+            return Response.FromValue(Pet.FromResponse(response), response);
```

Comment:
Why this changes?
Previously because the method name is different therefore we have to create another scope and now they are the same therefore we do not need the extra scope and just using the same scope in the protocol method?

Reply:
Yes, by design.